client/my-sites/drafts/author-selector.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import NavSegmented from 'components/section-nav/segmented';
+import NavItem from 'components/section-nav/item';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+function AuthorSelector( { onChange, selectedScope, translate, user } ) {
```

Comment:
We're not really selecting an author. Do you think `DraftsAuthorFilter` would be a better name? My thinking with prefix is to avoid potential confusion with [the other `<AuthorSelector />`](https://github.com/Automattic/wp-calypso/tree/master/client/blocks/author-selector).